data_pipelines/assets/__init__.py

```diff
@@ -1,16 +1,16 @@
 from dagster import load_assets_from_package_module
 
-from . import basins, deforestation, flood
+from . import basin, deforestation, flood
 
 DEFORESTATION = "deforestation"
-BASINS = "basins"
+BASIN = "basin"
```

Comment:
What is the point of these constants if they're only used once anyway? If that's the case you might as well just inline them